### ISSUE TITLE
Error when using crate because of error in dependency

### ISSUE THREAD

**Issue by freddd** [pdf-rs/pdf#75]
Hi, I'm getting an error when trying to use this crate.
```
❯ cargo run
   Compiling pdf_derive v0.1.20
error[E0433]: failed to resolve: could not find `export` in `syn`
  --> /Users/x/.cargo/registry/src/github.com-1ecc6299db9ec823/pdf_derive-0.1.20/src/lib.rs:99:23
   |
99 | type SynStream = syn::export::TokenStream2;
   |                       ^^^^^^ could not find `export` in `syn`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `pdf_derive`
```

This seems to be a suggested fix: https://github.com/frondeus/test-case/issues/60#issuecomment-755228907

**Comment by s3bk** [comment]
Hello,

looks like you just have to update the git repo: https://github.com/pdf-rs/pdf/commit/860d5c0315150e59f8e885a4ce98d1cfa0211db5
@MarkusPettersson98 fixed this a week ago.

**Comment by freddd** [comment]
Alright, thanks @s3bk - are you planning on publishing a new version of https://crates.io/crates/pdf?

**Comment by s3bk** [comment]
@freddd I just published pdf 0.7.1 including pdf_derive 0.1.21
